components/core/ApplicationUserControls.js

```diff
@@ -103,14 +103,21 @@ const STYLES_ITEM_BOX = css`
 `;
 
 const OpenIntercom = ({ user }) => {
-  const { show } = useIntercom();
+  const { show, update } = useIntercom();
   
-  const showWithProps = () => show({ 
-    name: user.data.name || user.username, 
-  });
-
   return(
-    <span style={{ cursor: "pointer" }} onClick={() => showWithProps()}>
+    <span 
+      style={{ cursor: "pointer" }} 
+      onClick={() => {
+          update({
+            name: user.data.name,
+            username: user.username,  
```

Comment:
Also, it looks like username is not being passsed through properly.  Can you make sure username and ideally the user's id is passed through? (not to be mistakened for the id that intercom generates for the user) 

I already added it on the intercom side as a custom data attribute like this, you just need to figure out how to pass them in: https://www.intercom.com/help/en/articles/179-send-custom-user-attributes-to-intercom
![image](https://user-images.githubusercontent.com/33686587/123488551-0428fa00-d5c5-11eb-91b0-d105fc3bf348.png)
![image](https://user-images.githubusercontent.com/33686587/123488584-173bca00-d5c5-11eb-9965-50f1d4d2adc1.png)